src/main/java/games/strategy/util/MD5Crypt.java

```diff
@@ -227,6 +219,22 @@ public static String crypt(final String password, String salt, final String magi
     return result.toString();
   }
 
+  /**
+   * Creates a new random salt that can be passed to {@link #crypt(String, String)}.
+   *
+   * @return A new random salt; never {@code null}.
+   */
+  public static String newSalt() {
+    final StringBuffer salt = new StringBuffer();
```

Comment:
Good point.  I just did an automatic Extract Method here and wasn't really paying attention to the actual code.